src/NCronJob/Configuration/Builder/JobOption.cs

```diff
@@ -17,7 +17,7 @@ internal sealed class JobOption
     /// <summary>
     /// The timezone that is used to evaluate the cron expression. Defaults to UTC.
     /// </summary>
-    public TimeZoneInfo TimeZoneInfo { get; set; } = TimeZoneInfo.Utc;
+    public TimeZoneInfo? TimeZoneInfo { get; set; }
```

Comment:
WHat would it mean if we don't have a TimeZOne?

Reply:
`GetNextCronOccurrence()` will select Utc.

![image](https://github.com/user-attachments/assets/a9855ad7-b8bb-49d4-9869-60df5e3bb1b2)